jwst/wfs_combine/wfs_combine.py

```diff
@@ -139,8 +139,9 @@ def create_aligned_2(self):
             # 1a. create image to smooth by first setting bad DQ pixels equal
             #     to mean of good pixels
             data_1 = self.input_1.data.astype(np.float64)
-            wh_bad_1 = np.where(self.input_1.dq != 0)
-            wh_good_1 = np.where(self.input_1.dq == 0)
+            wh_bad_1 = np.where(np.bitwise_and(self.input_1.dq, dqflags.pixel['DO_NOT_USE']) == 1)
```

Comment:
Travis is blowing up on these statements. You don't need the `==1` qualification, because `np.bitwise_and()` already equates to 1=True for pixels that satisfy the condition. All you need is:

wh_bad_1 = np.where(np.bitwise_and(self.input_1.dq, dqflags.pixel['DO_NOT_USE']))